.travis.yml

```diff
@@ -64,6 +64,7 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[all]"
+  - "./examples/app-recreate.sh"
```

Comment:
@hachreak @jirikuncar Another thought: the example app is tiny, and adding several extra shell scripts (`app-setup.sh`, `app-teardown.sh`, `app-fixtures.sh`, ...) may seem to overly complicate it.
What about having a single `examples/Makefile` file containing various setup, initdb, populate, teardown, run etc targets? Clean and easy... Reminded me of pypa's warehouse:
- https://github.com/pypa/warehouse/blob/master/Makefile